src/correlationUtils.ts

```diff
@@ -0,0 +1,118 @@
+import { getPearsonCorrelation, getPointBiserialCorrelation } from "./analyses";
+import { Correlations, DataType } from "./interfaces";
+import * as fs from 'fs';
```

Comment:
I did a quick search and couldn't find where this import was being used. If we don't **need** it, best to remove because I don't think `fs` is supported on Obsidian mobile

Reply:
Ah, you're right, that needs to be removed. It was part of me trying to generate test data.